paymentsheet-example/src/main/res/values/strings.xml

```diff
@@ -1,12 +1,31 @@
-<resources xmlns:tools="http://schemas.android.com/tools">
-    <string name="app_name" tools:ignore="MissingTranslation">PaymentSheet Example</string>
-    <string name="action_settings" tools:ignore="MissingTranslation">Settings</string>
-    <string name="product_subtitle" tools:ignore="MissingTranslation">1 x Regular %s</string>
-    <string name="subtotal" tools:ignore="MissingTranslation">Subtotal</string>
-    <string name="sales_tax" tools:ignore="MissingTranslation">Sales tax (8.25%)</string>
-    <string name="total" tools:ignore="MissingTranslation">Total</string>
-    <string name="hot_dog" tools:ignore="MissingTranslation">Hot Dog</string>
-    <string name="salad" tools:ignore="MissingTranslation">Salad</string>
-    <string name="cart_title" tools:ignore="MissingTranslation">Your cart</string>
-    <string name="buy" tools:ignore="MissingTranslation">Buy</string>
-</resources>
\ No newline at end of file
+<resources xmlns:tools="http://schemas.android.com/tools" tools:ignore="MissingTranslation">
+    <string name="app_name">PaymentSheet Example</string>
+    <string name="paymentsheet">PaymentSheet</string>
+    <string name="paymentsheet_custom">PaymentSheet Custom</string>
+    <string name="playground">Playground</string>
+    <string name="action_settings">Settings</string>
+    <string name="product_subtitle">1 x Regular %s</string>
+    <string name="subtotal">Subtotal</string>
+    <string name="sales_tax">Sales tax (8.25%)</string>
+    <string name="total">Total</string>
+    <string name="hot_dog">Hot Dog</string>
+    <string name="salad">Salad</string>
+    <string name="cart_title">Your cart</string>
+    <string name="buy">Buy</string>
+    <string name="customer">Customer</string>
+    <string name="customer_guest">Guest</string>
+    <string name="customer_new">New</string>
+    <string name="customer_returning">Returning</string>
+    <string name="google_pay">Google Pay</string>
+    <string name="on">On</string>
+    <string name="off">Off</string>
+    <string name="checkout_mode">Checkout Mode</string>
+    <string name="payment">Payment</string>
+    <string name="setup">Setup</string>
+    <string name="reload_paymentsheet">Reload PaymentSheet</string>
```

Comment:
should this be just called `prepareCheckout` or  more elaborated like "createPaymentIntent`?